e2e/app/page/create-account-page.ts

```diff
@@ -52,14 +54,16 @@ export default class CreateAccountPage extends BasePage {
   }
 
   async isLoaded(): Promise<boolean> {
-    await this.waitForTextExists('Sign In');
-    return true;
-  }
-
-  async getInvitationKeyInput(): Promise<Textbox> {
-    const textbox = new Textbox(this.page);
-    await textbox.withCss('#invitationKey');
-    return textbox;
+    try {
+      await Promise.all([
+        waitForText(this.page, 'Please read through the entire agreement to continue'),
+        this.page.waitForXPath('//*[@data-test-id="account-creation-tos"]', {visible: true}),
+      ])
```

Comment:
can you try this like:
Promise.all([...])
.then(values => { 
  return true
})
.catch(error => { 
  console.error(Message);return false
});

Reply:
eg  with await async function f() {

  let promise = new Promise((resolve, reject) => {
    setTimeout(() => resolve("done!"), 1000)
  });

  let result = await promise; // wait until the promise resolves (*)

  alert(result); // "done!"
}